autopts/pybtp/defs.py

```diff
@@ -55,19 +55,20 @@ def BIT(bit):
 BTP_SERVICE_ID_PBP = 0x1e
 # GENERATOR append 1
 
+BTP_ERROR = 0x00
```

Comment:
This seems unused, also defining `ERROR` to `0x00` seems quite atypical as `0` is more often than not used for success.

Reply:
"As general rule of thumb, the opcode for command matches the opcode for a response. Or the opcode 0x00 for an error is returned"
Please see overview.txt
But you are correct that this variable is unused

Reply:
Ah, right. Saw this the other day in the logs as well. If kept, it should be rename to something like `BTP_ERROR_OPCODE` as `BTP_ERROR` is very generic